app/controllers/users_controller.rb

```diff
@@ -33,6 +49,20 @@ def accept_screen(method, path)
     end
   end
 
+  def next_state
+    case user_state
+    when "new"
+      "transitioned"
+    when "transitioned"
+      "rolled_over"
+    when "rolled_over"
+      "features_accepted"
+    else
+      # go back to previous page with errors?
+      raise "unknown state"
```

Comment:
I'm assuming it's not likely this branch would be reached but if it did what should the behaviour be?